### DISCUSSION TITLE
My Computer isnt detecting Git

### DISCUSSION THREAD

**Discussion by mysterylawn** [community/community#23159] [New to GitHub]
<p>When I try to open a repository in GitBash, it tells me I don’t have it installed (which I do).<br>
I tried uninstalling and reinstalling both Git and Github, but to no avail. I would be very grateful if someone would help me.</p>

**Comment by byrneh** [comment]
<p>HI <a class="mention" href="/mysterylawn">@mysterylawn</a>, it is not exactly clear what you have done and what commands you are executing.<br>
It would help if you provide more details on steps to produce and error messages you see.<br>
This will then help us to help you.</p>
<p>Installing GitBash, then doing a git clone of a remote repository, then being able to work locally on the cloned local repository shouldn’t be a problem.</p>

**Comment by mysterylawn** [comment]
<p>I’m not currently doing anything of use, but GitHub is not detecting Git on my PC, which is concerning.<br>
Also, I just realized that if I open windows command prompt and try to use a git command it doesnt work either.</p>

**Comment by mysterylawn** [comment]
<p>Basically my computer isnt detecting Git, so i suppose I should update the title of this to such.</p>

**Comment by mysterylawn** [comment, marked as answer]
<p>nevermind, I figured it out. when I installed git on my new PC, I forgot to check the option in the installer for allowing git to access windows command prompt. whoops. I guess I was confused.</p>

**Comment by mysterylawn** [comment]
<p>thanks for your help and concern though! :slightly_smiling_face:</p>
